pure_changelog.md

```diff
@@ -7,6 +7,7 @@
 * Separated system volume from Bluetooth device volume for A2DP
 
 ### Fixed
+* Fixed update slider mode after tethering is off
```

Comment:
isn't that a regression fix? if so, I don't think if the entry is necessary

Reply:
Yes, it is. I just thought that we need to add everything to the changelog according to the development even if it is a regression :P
I'm about to remove that.